site/content/en/docs/manual/advanced/ai-tools.md

```diff
@@ -1,155 +1,309 @@
+## AI Tools
 
-- Click `Interact` to enter the interaction mode. Depending on the selected model,
-  the method of markup will also differ.
-  Now you can place positive and/or negative points. The [IOG](#inside-outside-guidance) model also uses a rectangle.
-  Left click creates a positive point and right click creates a negative point.
-  After placing the required number of points (the number is different depending on the model),
-  the request will be sent to the server and when the process is complete a polygon will be created.
-  If you are not satisfied with the result, you can set additional points or remove points.
-  To delete a point, hover over the point you want to delete, if the point can be deleted,
-  it will enlarge and the cursor will turn into a cross, then left-click on the point.
-  If you want to postpone the request and create a few more points, hold down `Ctrl` and continue (the `Block`
-  button on the top panel will turn blue), the request will be sent after the key is released.
+**AI tools** are available from the [controls sidebar](/docs/manual/basics/controls-sidebar/).
+<br>Click ![Magic wand](/images/image189.jpg) to start annotate.
 
-  ![](/images/image188_detrac.jpg)
+### Interactors
 
-- In the process of drawing, you can select the number of points in the polygon using the switch.
+Use interactors, to create a polygon semi-automatically.
 
-  ![](/images/image224.jpg)
+Supported DL models are not bound to the label, you can use them to track any object.
+
+When creating a polygon, you can use positive (regular) points
+or negative points (for some models):
+
+- Positive points are the points related to the object.
+- Negative points should be placed outside the object.
+
+![](/images/image188_detrac.jpg)
+
+In most cases specifying positive points is enough to build a polygon.
+
+**Adding points**
+
+To annotate with interactors, do the following:
+
+1. Click **Magic wand** ![Magic wand](/images/image189.jpg), and go to the **Interactors** tab.
+2. From the **Label** drop-down, select a label for the polygon.
+3. From the **Interactor** drop-down, select a model (see [Interactors Models](#interactors-models)).
+   <br>Click the **Question mark** to see information about each model:
+   <br>![](/images/image114_detrac.jpg)
+4. (Optional) If the model returns masks, and you need to
+   convert masks to polygon, use the **Convert masks to polygons** toggle.
+5. Click **Interact**.
+6. Use the left click to add positive points and the right click to add negative points.
+   <br>Number of points you can add depends on the model.
+7. On the top menu, click **Done** (or **Shift+N**, **N**), and move to the next frame.
 
-- You can use the `Selected opacity` slider in the `Objects sidebar` to change the opacity of the polygon.
-  You can read more in the [Objects sidebar](/docs/manual/basics/objects-sidebar/#appearance) section.
+When you place the points, the request is sent to the server immediately.
```

Comment:
Each algorithm has a minimum amount of points to annotate (indeed the input data for interactors can be different). Thus when the user places the required amount of points, the request will be sent automatically to the server. If we understand that the object is complex and we need more points by default, we can use CTRL to add more points in one request.